docs/_markbind/navigation/userGuideSections.md

```diff
@@ -0,0 +1,12 @@
+<markdown class="website-content">
+* ## Sections
```

Comment:
Might be more appropriate to say "User Guide" instead of "Sections" since this is now used everywhere.